.github/workflows/create-tests-matrices/action.yml

```diff
@@ -0,0 +1,48 @@
+inputs:
+    language-name:
+        description: "Language name"
+        required: true
+        default: false
```

Comment:
Please remove the default value. `false` is not valid language

Reply:
Good to know